#request
string go
---
#response
string[] trajectory_header
string[] trajectory_data
